packages/mx-adapter/src/adapter.ts

```diff
@@ -124,28 +125,10 @@ export class MxAdapter implements WidgetAdapter {
     return mapCredentials(res.data);
   }
 
-  async CreateConnection(
-    request: CreateConnectionRequest,
-    userId?: string,
-  ): Promise<Connection> {
+  #GetNewMemberPropertiesFromRequest(request: CreateConnectionRequest) {
```

Comment:
I learned that javascript allows private methods. The `#` makes it private. I did this because these methods still use the class properties but they are only used by other class functions. And so I don't have to write individual tests for each function, but they are still tested through the public functions which use them.

Reply:
The `CreateConnection` function was getting really big and complex so I made these other private functions in an attempt to abstract logic into more readable blocks of code.